extension/test/e2e/klarna.spec.js

```diff
@@ -20,7 +20,7 @@ const {
 } = require('../../src/config/constants')
 
 // Flow description: https://docs.adyen.com/payment-methods/klarna/web-component#page-introduction
-describe('::klarnaPayment::', () => {
+describe.skip('::klarnaPayment::', () => {
```

Comment:
why?

Reply:
It failed always before, but now it has been fixed in master branch.
https://github.com/commercetools/commercetools-adyen-integration/pull/826/commits/fe6356403e1c4e8ec1320ad845ea68d6dcc06bed